rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2017-08-16
```

Comment:
On our fork of libtock-rs the latest working compiler version is {{nightly-2017-12-16}}. If we pick a version above, {{xargo}} stops working. I guess, the problem is that {{xargo}} is relying on some specific internal behavior of {{cargo}}.

Reply:
We now fixed the reliance on this specific version on our fork. Moreover, this also fixes #12 which likely comes from incompatibility of the new version of the allocator with the provided nightly.